doc/source/faq.rst

```diff
@@ -0,0 +1,76 @@
+.. _faq:
+
+==========================
+Frequently Asked Questions
+==========================
+
+.. contents:: Contents
+   :local:
+
+
+Inverse Solution
+================
+
+How should I regularize the covariance matrix
```

Comment:
I think this should actually end with a question mark, I think most FAQs do that